Makefile

```diff
@@ -239,4 +208,24 @@ deploy-tektoncd-dashboard: tektoncd-dashboard-helm
 		--values ./deploy/tektoncd/dashboard/values.yaml \
 		--namespace $(TEKTON_NS)
 
-dev-deploy: deploy-nginx deploy-arangodb deploy-kibana deploy-mongodb deploy-pg deploy-tektoncd-pipeline deploy-tektoncd-dashboard
+deploy-dracon-dev: deploy-elasticoperator
+	make draconctl-image-publish CONTAINER_REPO=localhost:5000/ocurity/dracon
+	@helm upgrade dracon ./deploy/dracon/ \
+	 	  --install \
+		  --values ./deploy/dracon/values.dev.yaml \
+		  --create-namespace \
+		  --namespace $(DRACON_NS) \
+		  --set "global.image.draconVersion=$(DRACON_VERSION)"
+		  --wait
+
+dev-deploy: 
```

Comment:
you can simplify this to `dev-deploy: deploy-nginx deploy-arangodb deploy-tektoncd-pipeline deploy-tektoncd-dashboard deploy-dracon-dev`